processor/redactionprocessor/processor_test.go

```diff
@@ -133,11 +133,13 @@ func TestRedactSummaryDebug(t *testing.T) {
 	outTraces := runTest(t, allowed, redacted, masked, ignored, config)
 
 	attr := outTraces.ResourceSpans().At(0).ScopeSpans().At(0).Spans().At(0).Attributes()
-	var deleted []string
+	deleted := make([]string, len(redacted))
```

Comment:
Let's just do 
```suggestion
	deleted := make([]string, 0, len(redacted))
```

and we can keep changes to a minimum then